index.html

```diff
@@ -0,0 +1,197 @@
+            <section>
+            <h2>Title</h2>
+            </section>
+        </div>
+        <div class="content red">
+            
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div> 
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div>
+            </article>
+
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+        </div>
+
+        <div class="blue ad"></div>
+
+        <div class="text">
+            <p> See all results </p>
+        </div>
+
+        <div class="bigger-ad">
+            <div class="center-title"></div>
+            <div class="text-under"></div>
+            <div class="blue-button learn-more"></div>
+        </div>
+
+        <div class="person-of-the-week">
+            <div class="profile"></div>
+            <div class="text-description"></div>
+        </div>
+        <div class="six-articles">
+            <div class="articles-row-1">
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>            
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>
+
+            </div>
+
+            <div class="articles-row-1">
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>            
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>
+                <article class="six">
+                    <div class="profile-six"></div>
+                </article>
+
+            </div>
+        </div>
+        <div class="systems"></div>
+        <div class="guides">
+            <article class="story-2">
+                <h2 class="title-article"></h2>
+            </article>
+            <article class="story-2">
+                <h2 class="title-article"></h2>
+            </article>
+            <article class="story-2">
+                <h2 class="title-article"></h2>
+            </article>
+            
+        </div>
+
+        <div class="newsletter">
+            <div class="guide">
+                <article class="newsletter-article">
+                    <h2 class="newsletter-title-article"></h2>
+                </article>
+            </div>
+            <div class="red-square news">
+                <div class="blue-button learn-more subscribe"></div>
+            </div>
+        </div>
+        <div class="footer">
+
+        </div>
```

Comment:
Let us improve semantics by using ```<footer>``` instead of ```<div class="footer">```. Also, the footer has centered  links
![kr6](https://user-images.githubusercontent.com/46388113/74501609-9c4ba500-4ee2-11ea-9259-c97fecfe2749.PNG)